### PR TITLE
Vanessa/favorites fix

### PR DESCRIPTION
**Pull request by vahurtad**
- fixed lost changes from last PR
- fixed conflicts
- adding Favorites UI

### CODE REVIEW

File: src/candidate/redux/candidates.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Selectors
 
-export const getCandidate = (state, id) => state[CANDIDATE_NAMESPACE][id];
+export const getCandidate = (state, id) =>(state[CANDIDATE_NAMESPACE][id])
```
Comment (gmferland) [inline src/candidate/redux/candidates.js]: This seems dubious

**Comment by gmferland** [comment]
Yarn lock should not be deleted! Unless you want to have a conversation about using npm instead

**Review by gmferland — CHANGES_REQUESTED** [review]

**Comment by cgalbiati** [comment]
I think this still has duplicated commits (and maybe some that do not belong).  @gmferland I'm guessing that the deletion was a merge conflict issue, and not an attempt to stop using yarn

**Comment by gmferland** [comment]
Yeah, I actually would love to get rid of yarn. I think the latest version of npm is much better. But that's a convo for a different day

**Comment by vahurtad** [comment]
Hi guys, I think I have the correct changes done now with no conflicts on my end.

**Review by gmferland — APPROVED** [review]

**Review by marcaaron — APPROVED** [review]

**Review by cgalbiati — APPROVED** [review]

**Pull request merged**